fdbclient/ClusterInterface.h

```diff
@@ -73,6 +76,7 @@ struct ClusterInterface {
 		           getClientWorkers,
 		           forceRecovery,
 		           moveShard,
+		           splitShard,
```

Comment:
New interface needs to be appended to the end in order to be protocol compatible.

Reply:
Thanks for the tip, change applied.